llvm-bazel/llvm-project-overlay/mlir/tblgen.bzl

```
@@ -324,6 +324,8 @@ def gentbl(
       test: whether to create a shell test that invokes the tool too.
       **kwargs: Extra keyword arguments to pass to all generated rules.
     """
+    llvm_project_execroot_path = Label("//llvm:tblgen.bzl", relative_to_caller_repository = False).workspace_root
```

Nit: let's use this file's name instead

```suggestion
    llvm_project_execroot_path = Label("//mlir:tblgen.bzl", relative_to_caller_repository = False).workspace_root
```